### ISSUE TITLE
Minor accessibility criteria should be added to DataTables plugin examples

### ISSUE THREAD

**Issue by CoolColin** [wet-boew/wet-boew#8998]
Could this plugin working example be updated to incorporate an additional sufficient technique for data tables under WCAG? Currently the tables on this working example page have no designated/associated title/heading directly related to them. All data tables should really have a <caption> element especially on a page like this working example where there are more then 1 table on the page. 

Add:
HTML <caption> element. 

Captions play a vital role in labelling the table with an appropriate title/heading that stays with the table regardless of what way a user is navigating/using the page (tables mode is one). This could provide further reinforcement that all data tables should in fact be using the HTML <caption> element with every data table. (https://www.w3.org/WAI/WCAG21/Techniques/html/H39)

**Comment by thekodester** [comment]
This was discussed before: https://github.com/wet-boew/wet-boew/issues/7729
